### PR TITLE
Introduce Barista's considerations that speed up the testing development

### PR DESCRIPTION
**Pull request by rocboronat**
Related with #7 
Waiting for #51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,12 @@ public void some_important_test() throws Exception {
 }
 ```
 
+## Magic that Barista does for you
+
+In order to speed up testing, Barista keeps in mind some considerations.
+- **Scrolls when needed**: Filling a form inside a `ScrollView` requests you to scroll to the next item several times to avoid interacting with undisplayed widgets. To keep tests concise, Barista scrolls automatically before clicking a `Button` and before writing into an `EditText`.
```
Comment (Sloy) [inline README.md]: > before clicking a Button

Actually, clicking anything. This sentence might lead someone to think that it only works for buttons. Suggestion: Before clicking any View.

Also, I think it doesn't say clear enough how shitty is scrolling in Espresso.Here's an idea:
_"Interacting with Espresso in a `ScrollView` requires you to scroll to each view, which sometimes doesn't work the first time. Also trying to scroll outside a ScrollView makes produces an Exception, forcing you to change the test depending on the layout. To keep tests simpler. Barista scrolls....."_

It's probably too much text. But scrolling in Espresso is **so frustrating** that I think a small sentence doesn't give Barista enough credit.

File: README.md [hunk 2]
```diff
@@ -154,6 +154,12 @@ public void some_important_test() throws Exception {
...
+- **Just interacts with displayed Views**: Interacting with `Fragments` inside a `ViewPager` throws `AmbiguousViewMatcherException`, cos the widgets you interact with will be potentially repeated on different pages. Barista only interacts with displayed widgets, so you can focus on the behavior instead of wasting time on details.
```
Comment (Sloy) [inline README.md]: Little detail: It's not just about Fragments. Any ViewPager with any kind of View inside that have the same layout have the same problem.

Comment (rocboronat) [inline README.md]: That's true!

**Comment by rocboronat** [comment]
@Sloy please, feel free to improve this PR just committing into it. As your English is really better than mine, that will be easier for both 😄

File: README.md (hunk 1 above)
Comment (rocboronat) [inline README.md]: Wep! Right now, the autoscroll is just done with Buttons and EditTexts :·/

Comment (rocboronat) [inline README.md]: Oh! Both were wrong. As you mentioned, all methods do the magic, in exception of `BaristaRadioButtonActions`. Opening an issue called #54!

**Comment by rocboronat** [comment]
Wep! After having a lunch together... @Sloy 's suggestions applied verbatim!

**Pull request merged**